components/__tests__/fixtures/content/es/foo.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  foo: 'foo in español',
+  bar: '',
+  baz: null,
+  sections: [require('./bar')],
```

Comment:
I'm not quite following, could you elaborate?
I didn't include `baz` because it should fallback to the fallback `baz`.
`bar` is included to test the nested translation.